examples/code_review.py

```diff
@@ -34,18 +34,20 @@ def get_prs(repo):
     client = Client(host="http://{}".format(os.environ.get('DX_IP')))
     query_params = {
         "repo":repo,
-        "labels":"status: code review request",
-        "states":"OPEN"
-        }
-    response = client.github.prs.get(query_params=query_params)
+        "item_type":'pull_requests',
+        "labels[]":['status: code review request'],
+        "states[]":['OPEN'],
+        "limit[]":['first', '100']
```

Comment:
Looks like `states` and `limit` are usually the same. Should these be the default values if not supplied in the query params?

Reply:
This is true for limit and I will update the default accordingly, but I do like the default for states to be none, because in that case you get all issues/prs.